src/App.js

```diff
@@ -39,7 +39,7 @@ function App() {
         <a href="https://elizabethwarren.com/plans/ultra-millionaire-tax">Find out more about the wealth tax</a> and <a href="https://elizabethwarren.com/join-us">join the fight</a>
         <div className="disclaimer">
           <p>Not affiliated with the Warren For President campaign</p>
-          <p>Created by volunteers</p>
+          <p>Created by <a href="https://community.elizabethwarren.com/main/groups/33907/lounge">Tech for Warren</a>, a volunteer group. <a href="https://github.com/techforwarren/two-cent-stories/">Find this project on Github!</a></p>
```

Comment:
Can we send them over to the generic sign up page perhaps? https://elizabethwarren.com/join-us

I'm not sure getting more inbounds into our group is the most useful way we can use this traffic!

Reply:
I'd also kind of prefer, personally, to _not_ link to the repo. I defer to @joziesands but I've been avoiding that on my projects personally. I don't see the win in pointing people to the code here

Reply:
I don't think we should link the repo, and I agree that the most of the traffic will be best served by the EW join-us link. It is used on the page already, but I don't think having it twice would hurt, or maybe use events.elizabethwarren.com for one of them.

I'm very wary of all publicity specific to the group. It's nice that we haven't had to lock down our dev process. If this does get traction I think we'll gain more watchers(trolls or not) than volunteers for this group.